focus-ios/Blockzilla/Modules/WebView/LegacyWebViewController.swift

```diff
@@ -455,6 +455,12 @@ extension LegacyWebViewController: WKNavigationDelegate {
             break
         }
 
+        // Prevent Focus from opening deeplinks from links
+        if let scheme = navigationAction.request.url?.scheme, scheme == AppInfo.appScheme {
```

Comment:
Correct me if I'm wrong but I believe `scheme` is not required to be lower-cased, meaning that we should either do a case-insensitive string comparison here, or check against `scheme.lowercased()` etc.

```
(lldb) po url
▿ Optional<URL>
  ▿ some : hTtPs://mozilla.com
    - _url : hTtPs://mozilla.com
(lldb) po scheme
▿ Optional<String>
  - some : "hTtPs"
```

Reply:
Updated